k6-grafana/k6.libsonnet

```diff
@@ -19,9 +21,10 @@ local k6(name, parallelism) = {
             name: 'K6_STATSD_ADDR',
             value: 'k6-statsd.k6-operator-system.svc.cluster.local:8125',
           },
-        ],
+        ] + extraEnv,
       },
-    },
+    }
+    + cleanJobs
```

Comment:
you could make it a bit nicer with
```jsonnet
[if cleanup then 'cleanup' else null]: 'post'
```
I think

Reply:
Yep) Thanks